### PR TITLE
fix(autoloader): Also enforce up to date static autoloader

### PR DESCRIPTION
**Pull request by nickvergessen**
* Ref https://github.com/nextcloud/server/pull/44840/files#r1568928690

### CODE REVIEW

**Comment by nickvergessen** [comment]
/backport 3d8ba2bddbc03b6b5d64bf28361a962b7fbe1fef to stable29

**Comment by nickvergessen** [comment]
/backport 3d8ba2bddbc03b6b5d64bf28361a962b7fbe1fef to stable28

**Comment by nickvergessen** [comment]
/backport 3d8ba2bddbc03b6b5d64bf28361a962b7fbe1fef to stable27

**Review by AndyScherzinger — APPROVED** [review]

**Review by andrey18106 — APPROVED** [review]

File: build/autoloaderchecker.sh [hunk 1]
```diff
@@ -54,6 +54,11 @@ do
         composerfile=true
         break
     fi
+    if [[ $file == *autoload_static* ]]
```
Comment (nickvergessen) [inline build/autoloaderchecker.sh:57]: okay, so we are not checking that file as the dev dependencies make it dirty...

**Comment by nickvergessen** [comment]
Merged via https://github.com/nextcloud/server/pull/44886 already

**Pull request closed without merging**